components/Footer.tsx

```diff
@@ -33,9 +33,11 @@ export const FooterImpl: React.FC = () => {
     setHasMounted(true)
   }, [])
 
+  const currentYear  = new Date().getFullYear();
```

Comment:
this is probably fine.. but in general, injecting any date-related text into server-rendered pages can cause issues with react hydration

Reply:
ok，old way is better #476